code/lib/builder-vite/src/list-stories.ts

```diff
@@ -1,4 +1,4 @@
-import * as path from 'path';
+import * as posixPath from 'node:path/posix'; // Glob requires forward-slashes
```

Comment:
`import { join } from 'node:path/posix';`? not sure if it makes a difference in node

Reply:
Do you prefer the destructured version?  I slightly prefer the namespaced version, just because it's clear when it's being used where it's coming from.  `posixPath.join()` vs just `join()`.  But I don't feel strongly about it and happy to change if you'd like.

Reply:
Don't feel strongly. I think @ndelangen was making a push towards the namespaced version (but maybe only in browser code, for tree shaking?), so maybe he can comment here.